### ISSUE TITLE
gin-swagger   DATA RACE

### ISSUE THREAD

**Issue by locxiang** [swaggo/gin-swagger#112]
```
==================
WARNING: DATA RACE
Write at 0x00c00007f6c0 by goroutine 61:
  github.com/swaggo/gin-swagger.CustomWrapHandler.func1()
      /Users/xianghao/go/pkg/mod/github.com/swaggo/gin-swagger@v1.2.0/swagger.go:63 +0x169
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.RecoveryWithWriter.func1()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/recovery.go:83 +0x85
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.LoggerWithConfig.func1()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/logger.go:241 +0x1c5
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.(*Engine).handleHTTPRequest()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/gin.go:409 +0xa5d
  github.com/gin-gonic/gin.(*Engine).ServeHTTP()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/gin.go:367 +0x2f4
  net/http.serverHandler.ServeHTTP()
      /usr/local/go/src/net/http/server.go:2807 +0xce
  net/http.(*conn).serve()
      /usr/local/go/src/net/http/server.go:1895 +0x837

Previous write at 0x00c00007f6c0 by goroutine 63:
  github.com/swaggo/gin-swagger.CustomWrapHandler.func1()
      /Users/xianghao/go/pkg/mod/github.com/swaggo/gin-swagger@v1.2.0/swagger.go:63 +0x169
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.RecoveryWithWriter.func1()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/recovery.go:83 +0x85
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.LoggerWithConfig.func1()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/logger.go:241 +0x1c5
  github.com/gin-gonic/gin.(*Context).Next()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/context.go:161 +0xa6
  github.com/gin-gonic/gin.(*Engine).handleHTTPRequest()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/gin.go:409 +0xa5d
  github.com/gin-gonic/gin.(*Engine).ServeHTTP()
      /Users/xianghao/go/pkg/mod/github.com/gin-gonic/gin@v1.6.3/gin.go:367 +0x2f4
  net/http.serverHandler.ServeHTTP()
      /usr/local/go/src/net/http/server.go:2807 +0xce
  net/http.(*conn).serve()
      /usr/local/go/src/net/http/server.go:1895 +0x837

Goroutine 61 (running) created at:
  net/http.(*Server).Serve()
      /usr/local/go/src/net/http/server.go:2933 +0x5b6
  net/http.(*Server).ListenAndServe()
      /usr/local/go/src/net/http/server.go:2830 +0x102
  main.main()
      /Users/xianghao/go/src/zkjl/data-api/main.go:70 +0x61a

Goroutine 63 (running) created at:
  net/http.(*Server).Serve()
      /usr/local/go/src/net/http/server.go:2933 +0x5b6
  net/http.(*Server).ListenAndServe()
      /usr/local/go/src/net/http/server.go:2830 +0x102
  main.main()
      /Users/xianghao/go/src/zkjl/data-api/main.go:70 +0x61a
==================
```

**Comment by ubogdan** [comment]
Can you put a piece of code and the steps required to reproduce?

**Comment by ubogdan** [comment]
Closing: Fixed in #125.
